src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2083,29 +2084,37 @@ because xbuild doesn't support framework reference assemblies.
     ResolvedAssemblies="@(_ResolvedAssemblies)"
     ResolvedUserAssemblies="@(_ResolvedUserMonoAndroidAssemblies)"
     ErrorOnCustomJavaObject="$(AndroidErrorOnCustomJavaObject)"
-	ManifestTemplate="$(_AndroidManifestAbs)"
-	MergedManifestDocuments="@(ExtractedManifestDocuments)"
-	Debug="$(AndroidIncludeDebugSymbols)"
-	MultiDex="$(AndroidEnableMultiDex)"
-	NeedsInternet="$(AndroidNeedsInternetPermission)"
-	InstantRunEnabled="$(_InstantRunEnabled)"
-	AndroidSdkPlatform="$(_AndroidApiLevel)"
-	AndroidSdkDir="$(_AndroidSdkDirectory)"
-	PackageName="$(_AndroidPackage)"
-	ManifestPlaceholders="$(AndroidManifestPlaceholders)"
-	OutputDirectory="$(IntermediateOutputPath)android"
-	MergedAndroidManifestOutput="$(IntermediateOutputPath)android\AndroidManifest.xml"
+    ManifestTemplate="$(_AndroidManifestAbs)"
+    Debug="$(AndroidIncludeDebugSymbols)"
+    MultiDex="$(AndroidEnableMultiDex)"
+    NeedsInternet="$(AndroidNeedsInternetPermission)"
+    InstantRunEnabled="$(_InstantRunEnabled)"
+    AndroidSdkPlatform="$(_AndroidApiLevel)"
+    AndroidSdkDir="$(_AndroidSdkDirectory)"
+    PackageName="$(_AndroidPackage)"
+    ManifestPlaceholders="$(AndroidManifestPlaceholders)"
+    OutputDirectory="$(IntermediateOutputPath)android"
+    MergedAndroidManifestOutput="$(IntermediateOutputPath)android\AndroidManifest.xml"
     UseSharedRuntime="$(AndroidUseSharedRuntime)"
-	EmbedAssemblies="$(EmbedAssembliesIntoApk)"
-	ResourceDirectory="$(MonoAndroidResDirIntermediate)"
-	BundledWearApplicationName="$(BundledWearApplicationPackageName)"
-	PackageNamingPolicy="$(AndroidPackageNamingPolicy)"
-	ApplicationJavaClass="$(AndroidApplicationJavaClass)"
-	FrameworkDirectories="$(_XATargetFrameworkDirectories);$(_XATargetFrameworkDirectories)Facades"
-	AcwMapFile="$(_AcwMapFile)"
-	SupportedAbis="@(_BuildTargetAbis)">
+    EmbedAssemblies="$(EmbedAssembliesIntoApk)"
+    ResourceDirectory="$(MonoAndroidResDirIntermediate)"
+    BundledWearApplicationName="$(BundledWearApplicationPackageName)"
+    PackageNamingPolicy="$(AndroidPackageNamingPolicy)"
+    ApplicationJavaClass="$(AndroidApplicationJavaClass)"
+    FrameworkDirectories="$(_XATargetFrameworkDirectories);$(_XATargetFrameworkDirectories)Facades"
+    AcwMapFile="$(_AcwMapFile)"
+    SupportedAbis="@(_BuildTargetAbis)">
   </GenerateJavaStubs>
 
+  <ManifestMerger
+    ToolPath="$(JavaToolPath)"
+    JavaOptions="$(JavaOptions)"
+    ManifestMergerJarPath="$(MSBuildThisFileDirectory)manifestmerger"
+    AndroidManifest="$(IntermediateOutputPath)android\AndroidManifest.xml"
+    OutputManifestFile="$(IntermediateOutputPath)android\AndroidManifest.xml"
+    LibraryManifestFiles="@(ExtractedManifestDocuments)"
+  />
+
```

Comment:
I suspect this will make the `PerformanceTests` I added fail, we might have to wait on this one to get past that: https://github.com/xamarin/xamarin-android/pull/3913